waku/protocol/v2/waku_historic_messages.nim

```diff
@@ -0,0 +1,73 @@
+import 
+  libp2p/protocols/protocol,
+  libp2p/protobuf/minprotobuf,
+  stew/results
+
+const
+  HistoricMessagesCodec = "/waku/history/1.0.0"
+
+type
+  Cursor* = ref object
+    before*: seq[byte]
+    after*: seq[byte]
+
+  Query* = ref object
+    origin*: int32
+    to*: int32
+    bloom*: seq[byte]
```

Comment:
This seems specific to v1

Reply:
Hmm, have we decided not to use bloom filters? if so @oskarth, how do we reduce meta data leakage to mailservers?